src/panoramic/cli/remote/writer.py

```diff
@@ -96,3 +108,28 @@ def delete_model(self, model: PanoModel):
                 f'Failed to delete model {model.model_name} in dataset {model.virtual_data_source}', exc_info=True
             )
             raise ModelWriteException(model.virtual_data_source, model.model_name).extract_request_id(e)
+
+    def write_field(self, field: PanoField):
+        """Write model on remote API."""
+        logger.debug(f'About to write field {field.slug} in dataset {field.data_source}')
+
+        remote_field = map_field_from_local(field)
+        try:
+            self.field_client.upsert_fields(company_slug=self.company_name, fields=[remote_field])
+        except requests.RequestException as e:
+            logger.debug(f'Failed to write field {field.slug} in dataset {field.data_source}', exc_info=True)
+            if e.response is not None and e.response.status_code == requests.codes.bad_request:
+                raise InvalidFieldException(e).extract_request_id(e)
+            raise FieldWriteException(field.data_source, field.slug).extract_request_id(e)
+        pass
```

Comment:
Forgotten pass?

Reply:
Fixed in d6ae14fdbb23cb206b496e93be417f8a999c0f1b.